js-intro.js

```diff
@@ -46,16 +70,38 @@ var stringWithVowels2 = "ILoveJavaScript"
 var notAString1 = true
 var notAString2 = 42
 
+//inform user if variable passed into vowel remove function is not a string
+var stringDeterminer = (test) => {
+    if (test === "boolean") {
+        return ("The variable is not a string it is a boolean.")
+    } else  {
+        return ("The variable is not a string it is a number.")
+    }
+    }
+console.log(stringDeterminer(typeof notAString1))
+console.log(stringDeterminer(typeof notAString2))
+
```

Comment:
How can we wrap this logic into the logic you already created in the question above? So that we only have to call one function in total.